JavaFiddle-ejb/src/test/java/test2.java

```diff
@@ -0,0 +1,115 @@
+import org.junit.*;
+import org.junit.Test;
+import ru.javafiddle.core.ejb.GroupBean;
+import ru.javafiddle.core.ejb.ProjectBean;
+import ru.javafiddle.core.ejb.UserBean;
+import ru.javafiddle.jpa.entity.*;
+
+import javax.ejb.embeddable.EJBContainer;
+import javax.naming.Context;
+import javax.naming.NamingException;
+import java.io.UnsupportedEncodingException;
+import java.security.NoSuchAlgorithmException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by mac on 14.03.16.
+ */
+public class test2 {
```

Comment:
Please create proper class names - class name should give high-level info about the class purpose.

+ Consider using checkstyle plugin in IDEA.